swaScraper.py

```diff
@@ -5,6 +5,128 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as EC
 
+def correct_date(date):
+    """
+    Converts the date format to one accepted by SWA
+
+    SWA form cannot accept slashes for dates and is in the format YYYY-MM-DD
+    :param date: Date string to correct
+    :return: Corrected date string
+
+    """
+    a, b, c = date.split("/")
+
+    if(len(a) == 4):
+        # Assumed format is year, month, day
+        return "%s-%s-%s"%(a, b, c)
+    else:
+        # Assumed format is month, day, year
+        return "%s-%s-%s" % (c, a, b)
+
+def correct_time_string(timeString):
+    """
+    Returns a valid string version of the time of departure/arrival based on
+    given argument value
+
+    If no-value is given, "ALL_DAY" will be returned
+
+    :param timeString: String to correct
+    :return: Corrected string value
+    """
+    if timeString == "BEFORE_NOON":
```

Comment:
This can be changed in myParser.py
Set `default='ALL_DAY'`

Reply:
You can set the default to 'ALL_DAY' in myParser.py, but the function is  written guarantees that regardless of the input given a valid result is given without any added cost.